bio/nanovar/wrapper.py

```
@@ -0,0 +1,44 @@
+
+import shutil
+import tempfile
+from pathlib import Path
+
+from snakemake.shell import shell
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+# Optional gap/filter BED (-f). Built-in names (hg19/hg38/mm10) can go via extra.
+filter_bed = snakemake.input.get("filter_bed", "")
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)